src/map/conquest_system.cpp

```diff
@@ -713,7 +713,7 @@ namespace conquest
             charutils::AddPoints(PChar, charutils::GetConquestPointsName(PChar).c_str(), points);
             GainInfluencePoints(PChar, points / 2);
         }
-        return 0; // added conquest points (пока не вижу в этом определенного смысла)
+        return 0; // added conquest points (I don't see any point in it yet)
```

Comment:
some of these can be deleted instead of translated. pretty much when it is non helpful like this one is, axe it.

past me is guilty of this in English BUT: comments should be guiding the reader one understanding the code, not simply talking to the reader. if its not doing that, 99% chance it can be removed.